file/fat32.lisp

```diff
@@ -1825,12 +1842,6 @@ Valid media-type ara 'FAT32   ' " fat-type-label)))
       (cond ((not abort)
              (multiple-value-bind (time date) (get-fat-time)
                (when (member (direction stream) '(:output :io))
-                 (write-file (fat-structure host)
-                             (partition host)
-                             (buffer-position stream)
-                             (fat host)
-                             (buffer stream)
-                             file-length)
```

Comment:
Where is the current dirty block (if any) written back when the file is closed?

Reply:
It is mostly managed by fs-write-block.